pkg/virt-operator/resource/generate/install/strategy.go

```diff
@@ -353,12 +355,34 @@ func getMonitorNamespace(clientset k8coresv1.CoreV1Interface, config *operatorut
 }
 
 func DumpInstallStrategyToConfigMap(clientset kubecli.KubevirtClient, operatorNamespace string) error {
+	// Clean up existing install strategy ConfigMaps
+	labelSelector := labels.NewSelector()
+	requirement, err := labels.NewRequirement(v1.InstallStrategyLabel, selection.Exists, nil)
+	if err != nil {
+		return fmt.Errorf("failed to create label selector requirement: %v", err)
+	}
+	labelSelector = labelSelector.Add(*requirement)
 
 	config, err := operatorutil.GetConfigFromEnv()
 	if err != nil {
 		return err
 	}
 
+	existing, err := clientset.CoreV1().ConfigMaps(config.GetNamespace()).List(context.Background(), metav1.ListOptions{
+		LabelSelector: labelSelector.String(),
+	})
+	if err != nil {
+		return fmt.Errorf("failed to list old install strategy configmaps: %v", err)
```

Comment:
if it can't be listed due to whatever reason, an error will be returned and the new install strategy won't be created. was that the intention?